clang/test/OpenMP/nvptx_teams_reduction_codegen.cpp

```diff
@@ -2953,7 +2953,7 @@ int bar(int n){
 // CHECK3-NEXT:    [[TMP8:%.*]] = getelementptr inbounds [[STRUCT__GLOBALIZED_LOCALS_TY_0:%.*]], ptr [[TMP4]], i32 [[TMP5]]
 // CHECK3-NEXT:    [[C:%.*]] = getelementptr inbounds [[STRUCT__GLOBALIZED_LOCALS_TY_0]], ptr [[TMP8]], i32 0, i32 0
 // CHECK3-NEXT:    [[TMP9:%.*]] = load i8, ptr [[TMP7]], align 1
-// CHECK3-NEXT:    store i8 [[TMP9]], ptr [[C]], align 4
```

Comment:
I'm not sure if this change is correct. My intuition says that the alignment was incorrect and this now fixes it, hopefully someone can comment.

Reply:
Similar to above, this affects only accesses to a static global. Even if the new alignment is lower than optimal, we should be able to reconstruct it from the allocation. This is fine, IMHO.